### ISSUE TITLE
Default pager (less) conflicts with \watch

### ISSUE THREAD

**Issue by steveh** [dbcli/pgcli#837]
## Description

When using the default configuration, when the `\watch` command is used with `less`, you are prompted to continue after each query, limiting its usefulness.

```
example> select * from example; \watch 1
...
SELECT 0
(END)
```

Setting the pager to `less -E` automatically quits `less`, allowing `watch` to continue as expected.

```
example> \pager less -E
PAGER set to less -E.
Time: 0.000s
example> select * from example; \watch 1
...
SELECT 0
Time: 0.002s
Waiting for 1 seconds before repeating
```

This is obviously easy enough for the end user to change, but perhaps it could be done automatically, or suggested in the documentation?

## Your environment

- [ ] Linux 4.9.70-25.242.amzn1.x86_64 x86_64 GNU/Linux
- [ ] PGCLI 1.8.1

**Comment by amjith** [comment]
Fixed in #876

**Comment by amjith** [comment]
If you'd like to try it out you can install directly from master as follows:

```
pip install -U https://github.com/dbcli/pgcli/archive/master.zip
```
